### PR TITLE
chore(icons): sync Icons pt.1

### PR DESCRIPTION
**Pull request by xanderantoniadis**

### CODE REVIEW

File: .turbo/turbo-validate.log [hunk 1]
```diff
@@ -0,0 +1,1173 @@
+
```
Comment (ThrasyvoulosKafasis) [inline .turbo/turbo-validate.log:1]: Should this file exist on the repo?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ThrasyvoulosKafasis) [inline .vscode/settings.json:1]: Should this file exist on the repo?

File: src/icons/IconsList/IconsList.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    navigator.clipboard.writeText(iconName);
+    setCopiedIcon(iconName);
+    setTimeout(() => {
+      setCopiedIcon(null);
+    }, 1000); // Reset copiedIcon state after 2 seconds
+  };
+
+  return (
+    <div>
+      <input
```
Comment (ThrasyvoulosKafasis) [inline src/icons/IconsList/IconsList.tsx:35]: It would be better to use the `Input` component

File: .turbo/turbo-validate.log (hunk 1 above)
Comment (xanderantoniadis) [reply .turbo/turbo-validate.log]: Nope, deleted [here](https://github.com/epignosis/gnosis/pull/316/commits/8dea69c48d3c7d8e948644be3022125bf083078f)!

File: src/icons/IconsList/IconsList.tsx (hunk 3 above)
Comment (xanderantoniadis) [reply src/icons/IconsList/IconsList.tsx]: Fixed [here](https://github.com/epignosis/gnosis/pull/316/commits/d86ab9709111bab3d6d70c23fdfc2abc06c78e97)

**Review by lkatmadas — APPROVED** [review]

**Review by ThrasyvoulosKafasis — APPROVED** [review]

**Pull request merged**